src/app/habits/createhabit.tsx

```diff
@@ -0,0 +1,238 @@
+import Divider from "@/src/components/Divider";
+import Icon from "@/src/components/Icon";
+import { Text, View } from "@/src/components/Themed";
+import DefaultColors from "@/src/constants/DefaultColors";
+import colors from "@/src/constants/colors";
+import {
+  IconArrowForwardUp,
+  IconCheck,
+  IconSelector,
+  IconX,
+} from "@tabler/icons-react-native";
+import { router } from "expo-router";
+import { useColorScheme } from "nativewind";
+import { useContext, useEffect, useState } from "react";
+import { TextInput, TouchableOpacity } from "react-native";
+import {
+  Menu,
+  MenuOption,
+  MenuOptions,
+  MenuTrigger,
+} from "react-native-popup-menu";
+import { IconContext } from "./_layout";
+import { IconButton } from "./icon-button";
+import { icons } from "./icons";
+
+export default function Createhabit({ habitid }: { habitid: String }) {
+  const { colorScheme } = useColorScheme();
+  const borderColor = DefaultColors[colorScheme].tint;
+  const { icon } = useContext(IconContext);
```

Comment:
I think use IconPhotoScan as the default instead, and make it grey like the placeholder text